### ISSUE TITLE
Use focus-ring mixin for autocomplete component

### ISSUE THREAD

**Issue by fateeand** [AbsaOSS/cps-shared-ui#635]
Currently, the focused state is indicated by borders. We should use a focus ring instead, which would also resolve the issue with borderless and underlined components where the active state is not visible due to the absence of borders.
